src/frontend/apps/e2e/__tests__/app-desk/team.spec.ts

```diff
@@ -16,6 +16,9 @@ test.describe('Team', () => {
       await createTeam(page, 'team-top-box', browserName, 1)
     ).shift();
 
+    await expect(page.getByText('Group members')).toBeVisible();
+    await expect(page.getByLabel('Filter member list')).toBeVisible();
```

Comment:
Could we add a test to check that the requests are correct ?

Reply:
Yes, I added them in ```src/frontend/apps/desk/src/features/teams/member-management/__tests__/MemberGrid.test.tsx```

Reply:
Yes but the tests e2e don't only test your front, it tests as well your backend, if you can, prefer e2e tests over unit tests, the test coverage is far bigger with much less code.